src/tools-support/release-notes/app-center-2019.markdown

```diff
@@ -88,3 +89,102 @@ By connecting your Concur Expense account with Parken+, you can easily submit yo
 
 Parken+ works with Expense for Concur Standard and Expense for Concur Professional. It will be available soon in the SAP Concur App Center under User Connections (formerly Apps for Me) at [https://www.concur.com/app-center](https://www.concur.com/app-center). For more information about the Parken+ integration, visit
 [http://www.parken.plus/](http://www.parken.plus/).
+
+## <a name="march"></a>March 2019
+
+* [Apps for Me / User Connection Apps Now Available](#apps-for-me-connection-now-available)
+  * [Kapten](#kapten)
+* [Apps for Me / User Connection Apps Coming Soon](#apps-for-me-connection-coming-soon)
+  * [prayerletters.com](#prayerletters.com)
+  * [Parken+](#parken+)
+
+### <a href="apps-for-me-connection-now-available"></a>Apps for Me / User Connection Apps Now Available
+
+#### <a href="kapten"></a>Kapten
+
+![Kapten Logo](./app-center-2019-03-14-kapten-logo.png)
+
+Make ordering transportation and processing expenses painless: connect Concur Expense and Kapten. Ride comfortably with the professional, quality service lauded by BFM as “The French leader in ride-hailing” and trusted by over 2 million loyal customers.
+
+Kapten is a ride-hailing service that allows you to order a car in just a few clicks. It is available in Paris, Lyon, the French Riviera and Lisbon.
+
+OUR INTEGRATION WITH CONCUR EXPENSE
+To end the hell of expense reports, connect Concur to Kapten!
+* Click on the link below to start
+* Create your Kapten account or connect to your existing account
+* When ordering, switch to your business account
+* You'll be able to consult all invoices in your Concur account
+* You'll be saving time: invoices are also attached!"
```

Comment:
Looks like there's an extra " at the end of the last bullet. Since the first and last have ending punctuation should the rest end in periods? Same comment about ending punctuation for the other lists. The Parken section has them.